cms/templates/library.html

```diff
@@ -58,12 +58,12 @@ <h3 class="sr">${_("Page Actions")}</h3>
             <ul>
                 <li class="nav-item">
                     <a href="#" class="button new-button new-component-button">
-                        <i class="icon fa fa-plus icon-inline"></i> <span class="action-button-text">${_("Add Component")}</span>
+                        <span class="icon fa fa-plus icon-inline" aria-hidden="true"></span> <span class="action-button-text">${_("Add Component")}</span>
                     </a>
                 </li>
                 <li class="action-item action-toggle-preview nav-item">
                     <a href="#" class="button button-toggle-preview action-button toggle-preview-button is-hidden">
-                        <i class="icon fa fa-refresh"></i>
+                        <span class="icon fa fa-refresh" aria-hidden="true"></span>
```

Comment:
If there is visible text, no sr text is necessary. The text for the action will be announced.